packages/serverpod_shared/lib/src/config.dart

```diff
@@ -100,11 +100,13 @@ class ServerpodConfig {
   factory ServerpodConfig.loadFromMap(
     String runMode,
     String serverId,
+    bool isServerIdDefault,
```

Comment:
Nit: Coming at this method from the usage below, I was at first confused what the literal true/false would mean.

Usually it's probably nicer to use a named parameter if the bool is one of many arguments.
Or could this be a small record, if the flag is really needed for each serverID?

(Again, this might also not matter much going forward, so take this with a grain of salt.)